README.md

```diff
@@ -71,6 +67,15 @@ Or through guard (`--no-interactions` allows the use of `pry` inside tests):
 bundle exec guard --no-interactions
 ```
 
+## Running specs in parallel
+
+When running specs in parallel for the first time you need to do the following:
+
+1. `bundle exec rails parallel:create` ...to create test databases.
+2. `bundle exec rails parallel:prepare` ... to run migrations.
+3. `bundle exec rails parallel:spec` ...to run specs in parallel using the maximum available cores.
+4. `bundle exec rails parallel:drop` ...to drop the test databases.
+
```

Comment:
so we need to go 1,2,3,4 then can continue with `bundle exec rake`?

Reply:
@defong - you need to do steps 1,2,3 to run the tests in parallel. 
This sets up X number of test databases depending on the number of cores on your machine. 
To drop these databases you need to run `bundle exec rails parallel:drop`

I will see if there is a way to automate 1 - 3

Reply:
@defong - Discovered you can actually set up parallel testing locally by running `bundle exec rake parallel:setup` - have updated README